components/brave_wallet/browser/zcash/rust/lib.rs

```diff
@@ -330,7 +399,7 @@ fn create_orchard_builder_internal(
         let _ = match Option::from(orchard::Address::from_raw_address_bytes(&out.addr)) {
             Some(addr) => {
                 builder.add_output(None, addr,
-                    orchard::value::NoteValue::from_raw(out.value), None)
+                    orchard::value::NoteValue::from_raw(u64::from(out.value)), None)
```

Comment:
Yes, we use u32 since it's enough(i had a conversation with zcash team about that. Initially they declared many fields as u64 in the proto file, but they are converted to u32 finally. Also they store them as u32 in the rusqlite db). C++ sqlite doesn't have native u64 support, so it is more convenient to use u32.